src/server/helpers/utils.js

```diff
@@ -52,13 +72,36 @@ function getEntityModelByType(type) {
 	return entityModels[type];
 }
 
+/**
+ * Regular expression for valid BookBrainz IDs
+ *
+ * @type {RegExp}
+ * @private
+ */
 const _bbidRegex =
 	/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/;
 
+/**
+ * Tests if a BookBrainz ID is valid
+ *
+ * @param {string} bbid - BookBrainz ID to validate
+ * @returns {boolean} - Returns true if BookBrainz ID is valid
+ */
 function isValidBBID(bbid) {
 	return _bbidRegex.test(bbid);
 }
 
+/**
+ * Helper function that allows the values of an object that
+ * is passed in at a later time to be interpolated into a
+ * string
+ *
+ * @param {string[]} strings - Array of string literals from template
+ * @returns {function(*)} - Takes an object/array as an argument.
+ * When invoked, it will return a string with all the key names from
+ * the tagged template literal replaced with their corresponding values
+ * from the newly passed in object.
+ */
 // Cribbed from MDN documentation on template literals
```

Comment:
If we can find a link for this documentation, it'd be good to include it in the JSDoc comment - @leftmostcat I think this came from you?